lib/java/com/google/android/material/checkbox/MaterialCheckBox.java

```diff
@@ -732,7 +732,7 @@ private void refreshButtonDrawable() {
     updateButtonTints();
 
     super.setButtonDrawable(
-        DrawableUtils.compositeTwoLayeredDrawable(buttonDrawable, buttonIconDrawable));
```

Comment:
Please keep the original method name. I believe it's being verbose for a reason.

Reply:
The method doesn't guarantee it always returns layered drawable, so, technically, the original name is incorrect, that's why I've changed it.

Reply:
I don't think the original method name implies that....

I think "two layered drawable" refers to the input values, not the output value. 

Another thing is restricted APIs are still technically "public". There can still be clients affected by this change. Let avoid unnecessary changes like this as much as possible.

Reply:
If you are right, the name is even more meaningless and incorrect. First, the method already takes two arguments, no need to indicate it also in the name. Second, it's not guaranteed that both drawings must be layered. And even if it were so, it would be enough to declare the input parameters types as `LayerDrawable` and this "name refinement" would again be redundant.

About `@RestrictTo`: Restricted APIs are `public` only because there is no equivalent of `internal` in Java. And if the client explicitly ignored the restrictions of some API, it should definitely have been prepared for the potential consequences.

I think it's better to fix the name instead of leaving it only because someone might use this "only for Material internal purposes" API.